### DISCUSSION TITLE
Allow string or numeric primary parameters on functions

### DISCUSSION THREAD

**Discussion by DarinClarkSE** [thunderclient/thunder-client-support#1295] [Ideas]
Currently to call a function with parameters which aren't environment variables I have to use
{{@|incrementNumber("NUM")}}

And the function looks like 
function incrementNumber(throwAway,variableName){
    var varValue = tc.getVar(variableName);
    varValue++;
    tc.setVar(variableName,varValue);
    return varValue;
}

It would be nice if that could be simplified to 
{{"NUM"|incrementNumber()}}
or 
{{incrementNumber("NUM")}}

And the function wouldn't need the throwAway parameter.

**Comment by rangav** [comment]
Hi @DarinClarkSE 

The below syntax is difficult to implement
`{{"NUM"|incrementNumber()}}
`or
`{{incrementNumber("NUM")}}`

But I can see if it's possible to remove `throwAway` parameter requirement easily
